2-mandatory/2-financial-times.js

```diff
@@ -6,36 +6,90 @@
 */
 function potentialHeadlines(allArticleTitles) {
     // TODO
+
+  let articleTitles = [];
+  for (let i = 0; i < allArticleTitles.length; i++) {
+    if (allArticleTitles[i].length <= 65) {
+      articleTitles.push(allArticleTitles[i]);
+    }
+  }
+  return articleTitles;
 }
 
+
 /*
     The editor of the FT likes short headlines with only a few words!
     Implement the function below, which returns the title with the fewest words.
     (you can assume words will always be seperated by a space)
 */
+
+
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+
+        // TODO
+
+  let shortestTitle = 0;
+
+//   Infinity initializes shortest Title Length to ensure any subsequent title with
+//    fewer words replaces the previous shortest title.
+  let shortestTitleLength = Infinity;
```

Comment:
this is cool :)

Reply:
I saw this on a YouTube channel called ELzero